### ISSUE TITLE
Missing links in Related Links for Export-SPOQueryLogs.md

### ISSUE THREAD

**Issue by sympmarc** [MicrosoftDocs/office-docs-powershell#1054]
There are two links at the bottom of the [article](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/export-spoquerylogs?view=sharepoint-ps); both are missing their URLs.

I believe the first one should be:
https://support.office.com/en-us/article/introduction-to-the-sharepoint-online-management-shell-c16941c3-19b4-4710-8056-34c034493429
but I'm not sure about the second.

**Comment by kenwith** [comment]
@sympmarc - thank you for pointing this out. I missed it when I published these. I will get this sorted out ASAP. Really appreciate the feedback!

**Comment by kenwith** [comment]
@sympmarc  - these are fixed now and it should be live anytime once caches expire. Here are the links below. Thank you for noticing and taking the effort to report!

[Introduction to the SharePoint Online management shell](https://docs.microsoft.com/en-us/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps)

[Set up the SharePoint Online Management Shell Windows PowerShell environment](https://docs.microsoft.com/en-us/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)

**Comment by kenwith** [comment]
By the way, you noticed a really good point because as we were researching we found other links too that need to be consolidated to avoid confusion. Those links are below and we will consolidate. We think the links I added to the article (in comment above) are source of truth and if you find anything else in any of these links below let me know and I will make sure it gets into the links above.

Here are the links we found: 

1.	https://support.office.com/en-us/article/introduction-to-the-sharepoint-online-management-shell-c16941c3-19b4-4710-8056-34c034493429
2.	https://docs.microsoft.com/en-us/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps
3.	https://docs.microsoft.com/en-us/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps
